scripts/coder-dev.sh

```diff
@@ -35,16 +37,20 @@ CODER_DEV_DIR="$(realpath ./.coderv2)"
 CODER_DELVE_DEBUG_BIN=$(realpath "./build/coder_debug_${GOOS}_${GOARCH}")
 popd
 
+if [ -n "${CODER_AGENT_URL}" ]; then
```

Comment:
We could also check if `CODER=true`.  That's what I do in my dotfiles.

Reply:
Should also maybe be `"${CODER_AGENT_URL:-}" in case the env var is not set, on account of the `set -u`.